src/config.ts

```diff
@@ -1,3 +1,6 @@
+import which from "which"
+import { existsSync } from "fs"
```

Comment:
sync causes blocking the main thread. we should use async since the function is alread async.

Reply:
We need the result of this right away. I don't think there is any difference between `await exists` and `existsSync`!

Reply:
`fs.access` is the async version of `fs.exists`